packages/core/src/loaders/filesystem/filesystemLoader.ts

```diff
@@ -12,7 +12,7 @@ export class FilesystemLoader {
    * There is no way to filter by a type in runtime :(
    */
   public async load(/*<Resource>*/ _opts?: IFilesystemLoaderOpts): Promise<IHttpOperation[]> {
-    const fsPath = _opts ? _opts.path : DEFAULT_PATH;
+    const fsPath = !_opts || !_opts.path ? DEFAULT_PATH : _opts.path;
```

Comment:
This line took me a minute to figure out. Maybe the inverse would be a little more clear:

```
const fsPath = _opts && _opts.path ? _opts.path : DEFAULT_PATH;
```